app/services/attribute_asserter.rb

```diff
@@ -64,13 +63,9 @@ def add_email(attrs)
     }
   end
 
-  def add_phone(attrs)
-    attrs[:phone] = { getter: :phone }
-  end
-
   def bundle
     @_bundle ||= (
-      authn_request_bundle || service_provider.metadata[:attribute_bundle] || DEFAULT_BUNDLE
+      authn_request_bundle || service_provider.metadata[:attribute_bundle] || []
```

Comment:
this is a significant change -- see other comment about LOA3 and default bundle.